s3torchconnectorclient/rust/src/lib.rs

```diff
@@ -27,6 +28,12 @@ mod put_object_stream;
 mod python_structs;
 mod build_info;
 
+#[pyfunction]
+#[pyo3(name = "enable_debug_logging")]
```

Comment:
Let's prefix this function with an underscore, and possibly make the name more obvious it's for CRT logging